modules/spark/src/test/scala/org/apache/ignite/spark/IgniteCatalogSpec.scala

```diff
@@ -39,9 +39,12 @@ class IgniteCatalogSpec extends AbstractDataFrameSpec {
         it("Should observe all available SQL tables") {
             val tables = igniteSession.catalog.listTables.collect()
 
-            tables.length should equal (3)
+            tables.length should equal(6)
 
-            tables.map(_.name).sorted should equal (Array("CITY", "EMPLOYEE", "PERSON"))
+            // each table is accessible both with and without schema; tables which were created via CREATE TABLE
+            // expressions reside in the PUBLIC schema whereas cache-created tables reside in their own schema
+            tables.map(_.name).sorted should equal(
```

Comment:
I think we should return each table only once.
Seems, we should use `schema.name` form to be sure we can use it in all cases.

Reply:
If we just use the `schema.name` form then we will break anyone who is currently using the catalog and depending on just the `name` form being in the tables list. The current change is backwards-compatible with current assumptions.

I'll take your advice on the best course of action here - do you want to break backwards-compatibility?

Reply:
> do you want to break backwards-compatibility?
> we will break anyone who is currently using the catalog and depending on just the `name`

Yes, I understand that we will break compatibility with this change.
Personally, I like to fix it now and live with the correct solution.

Let's discuss it on dev-list, so anyone can share an opinion.